### ISSUE TITLE
Loading template from string (Documentation)

### ISSUE THREAD

**Issue by lukeholder** [twigphp/Twig#3173]
I am following the recipe at:

https://twig.symfony.com/doc/2.x/recipes.html#loading-a-template-from-a-string

for loading a template from a string. The example code is the following:

```
$template = $twig->createTemplate('hello {{ name }}');
echo $template->render(['name' => 'Fabien']);
```

This works fine, but the PHPdoc for `createTemplate()` is: https://github.com/twigphp/Twig/blob/2.x/src/Environment.php#L433-L446

I am a little confused on these lines:

```
* @param string $template The template name
* @param string $name     An optional name of the template to be used in error messages
```

Yet the docs show passing in a whole template in the $template name param?

Also wondering if the other method mentioned in the recipe (`render()`) could be documented in the source? 
https://github.com/twigphp/Twig/blob/2.x/src/Template.php#L383

Thanks

**Comment by stof** [comment]
that's a mistake in the phpdoc. `$template` for this method is the template source, not the template name.

**Comment by stof** [comment]
> Also wondering if the other method mentioned in the recipe (`render()`) could be documented in the source?

The Template class is an internal one. The public API of Twig always return a TemplateWrapper object. And TemplateWrapper has full type information in its API (either through phpdoc or through PHP typehints)

**Comment by fabpot** [comment]
Fixed in 2024406e
